bin/i18n/sync-codeorg-out.rb

```diff
@@ -31,6 +34,80 @@ def rename_from_crowdin_name_to_locale
   end
 end
 
+# Distribute downloaded translations from i18n/locales
+# back to blockly, apps, pegasus, and dashboard.
+def distribute_translations
+  total_locales = Languages.get_locale.count
+  Languages.get_locale.each_with_index do |prop, i|
+    locale = prop[:locale_s]
+    print "Distributing #{locale} (#{i}/#{total_locales})\r"
+    $stdout.flush
+    next if locale == 'en-US'
+    next unless File.directory?("i18n/locales/#{locale}/")
+
+    ### Dashboard
+    Dir.glob("i18n/locales/#{locale}/dashboard/*.yml") do |loc_file|
+      relname = File.basename(loc_file, '.yml')
+      source = "i18n/locales/source/dashboard/#{relname}.yml"
+
+      # Special case the un-prefixed Yaml file.
+      destination = (relname == "base") ?
+        "dashboard/config/locales/#{locale}.yml" :
+        "dashboard/config/locales/#{relname}.#{locale}.yml"
+
+      if locale == 'hy-AM' # Armenian accepts English translations, does not need fallback
+        export_without_merge "yml", loc_file, destination
+      else
+        merge_translation "yml", source, loc_file, destination
+      end
+    end
+
+    ### Apps
+    js_locale = locale.tr('-', '_').downcase
```

Comment:
Would be great to do a one-time rename so we don't need to think about this special case.

Reply:
You mean like actually move the files? That would probably be a decently large work item